### PR TITLE
Feature/cleanup PR 

### PR DESCRIPTION
**Pull request by patrickmcclory**
Cleaning up a lot of little things: 

* destinationPath fix will default to ./{project name} if workingdir option not set at command line
* fix to formatting 
* some syntax fixes in bash scripts
* adding binary check fixes for chef/puppet stuff. 
* also fixing ps1 script for calling packer

### CODE REVIEW

File: generators/ci/index.js [hunk 1]
```diff
@@ -65,30 +65,43 @@ var BakeryCI = yeoman.Base.extend({
...
+        break;
+      default:
+        feedback.warn('CI toolset ' + ciInfo.createci +
+          ' is not currently available. Skipping CI script setup.');
+        break;
     }
     if (file != '') {
       this.fs.copyTpl(
         this.templatePath(file),
-        this.destinationPath(file),
```
Comment (infernokodiak) [inline generators/ci/index.js:85]: why are you removing this.destinationPath?

Comment (patrickmcclory) [reply generators/ci/index.js:85]: @dgillman - thoughts?

Comment (patrickmcclory) [reply generators/ci/index.js:85]: I'll pull those back into all of the file path calls.

Comment (patrickmcclory) [reply generators/ci/index.js:85]: addressed in 02b170e3ec4261ed8a29fe1b50e8e5b3045606ec

**Comment by patrickmcclory** [comment]
As of 4dd55b9 - working for puppet and chef, validated locally.

**Comment by infernokodiak** [comment]
👍

**Pull request merged**